src/toast/observation.py

```diff
@@ -94,6 +94,10 @@ def set_default_values(values=None):
         # Units
         #
         "det_data_units": u.Kelvin,
+        #
+        # Noise
+        #
+        "noise_model" : "noise_model",
```

Comment:
This is a good idea!  Now we can use that as the default trait value for many operators expecting or generating the default noise model.